app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -267,6 +267,8 @@ function handleCommentSubmit(event) {
               input.value = newComment.comment_template || "";
             });
 
+            commentInputs[1].style.height = null
```

Comment:
tiny change: could you add the trailing `;`?

Reply:
Missed that thanks